tests/api/test_discount.py

```diff
@@ -167,8 +168,8 @@ def test_sale_query(staff_api_client, sale, permission_manage_discounts):
 
 
 def test_create_voucher(staff_api_client, permission_manage_discounts):
-    start_date = date(day=1, month=1, year=2018)
-    end_date = date(day=1, month=1, year=2019)
+    start_date = timezone.now() - timedelta(days=365)
+    end_date = timezone.now() + timedelta(days=365)
```

Comment:
`start_date` and `end_date` are almost equal. Previously, the difference was 1 year.
Same commet for the rest of tests 👇